src/commands/hook/hook.js

```diff
@@ -4,7 +4,7 @@ const HOOK: Object = {
   PATH: '/hooks/prepare-commit-msg',
   CONTENTS:
     '#!/bin/sh\n# gitmoji as a commit hook\n' +
-    'exec < /dev/tty\ngitmoji --hook $1\n'
+    'exec < /dev/tty\ngitmoji --hook $1 $2\n'
```

Comment:
Why we need to pass a second arg?

Reply:
This second argument is there to access the [commit message source from git](https://git-scm.com/docs/githooks#_prepare_commit_msg), allowing gitmoji-cli to detect amending, merge, etc.